.github/workflows/main.yml

```
@@ -0,0 +1,43 @@
+#main.yml
+
+name: Main
+on: pull_request
```

Not sure if we should run on `push` as well.

---

Yea, couldn't hurt to build `main` as well to check for merge issues.
```suggestion
on:
  # Build main every time something is merged.
  push:
    branches: [ main ]
  # Build on all pull requests, regardless of target branch.
  pull_request:
```